apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -60,9 +56,17 @@ loop(Socket, Transport, State) ->
                     {ok, Ms} = metric:list(Bucket, Prefix),
                     Transport:send(Socket, dproto_tcp:encode_metrics(Ms)),
```

Comment:
Remove the dynamic function call on line 57. Only modules that define callbacks should make dynamic calls.